clients/ctl/admin-ui/src/features/common/AccordionTree.tsx

```diff
@@ -91,7 +110,11 @@ const AccordionTree = ({ nodes, onEdit, focusedKey }: Props) => {
             {node.label}
           </AccordionButton>
           {isHovered ? (
-            <ActionButtons node={hoverNode} onEdit={onEdit} />
+            <ActionButtons
+              node={hoverNode}
+              onEdit={onEdit}
+              onDelete={onDelete}
```

Comment:
The AccordionTree is nice and generic, other than this pass-through of the action callbacks. We could replace those callback props with a react child prop and then the consumer could decide what's shown on hover. Something like:
```ts
interface Props {
  nodes: TreeNode[];
  focusedKey?: string;
  renderHover?: (node: TreeNode) => React.ReactNode
}
const AccordionTree = ({ ... }: Props) => { ... }

// Usage
 <AccordionTree
  nodes={taxonomyNodes}
  focusedKey={editEntity?.fides_key}
  renderHover={(node) => <ActionButtons node={node} onEdit... } />
```

Reply:
ah! yeah I had been bothered by that, but didn't know a good pattern to work around it. thanks for the great suggestion 👍